system/adb/Kconfig

```diff
@@ -49,6 +49,16 @@ config ADBD_TOKEN_SIZE
 
 endif # ADBD_AUTHENTICATION
 
+if BOARDCTL_RESET
+config ADBD_RESET_RECOVERY
+	int "Reset argument for recovery"
+	default 1
```

Comment:
`1` is reserved for `EXIT_FAILURE`. Should we default `ADBD_RESET_RECOVERY` to `2` and `ADBD_RESET_BOOTLOADER` to `3`?

Reply:
Th argument pass to BOARDCTL_RESET is used to control the reboot target, not the error code, I think.